picard/tagger.py

```diff
@@ -229,6 +229,10 @@ def __init__(self, method_name, help_text=None, help_args=None):
         "handle_command_fingerprint",
         help_text="Calculate acoustic fingerprints for all (matched) files in the album pane.",
     ),
+    "LOG_CLEAR": RemoteCommand(
```

Comment:
Others remote commands start with a verb, so I think `CLEAR_LOGS` would be more consistent (and plural because it clears all logs).

Now I'm not sure we need to clear all logs in one command, we could also provide a generic command `CLEAR_LOG <logname>`, with `logname` being either  `main` or `history`.

@rdswift what's your take on this?

Reply:
That is likely better, but we'll need to provide some explanation somewhere about the difference between the two (because even I'm not sure :wink: ).

Reply:
Ok, let's keep it simple for now, we renamed the command `CLEAR_LOGS` and if we need in the future we can still introduce `CLEAR_LOG <logname>`.

@skelly37 can you make this change?

Reply:
> Others remote commands start with a verb, so I think `CLEAR_LOGS` would be more consistent (and plural because it clears all logs).

In that case, should we also be renaming the `FROM_FILE` command to something like `READ_COMMANDS_FROM_FILE` (or something a bit shorter, but still starting with a verb)?

Reply:
> Ok, let's keep it simple for now, we rename the command `CLEAR_LOGS` and if we need in the future we can still introduce `CLEAR_LOG <logname>`.
> 
> @skelly37 can you make this change?

Just rename into `CLEAR_LOGS`?

Reply:
> > Others remote commands start with a verb, so I think `CLEAR_LOGS` would be more consistent (and plural because it clears all logs).
> 
> In that case, should we also be renaming the `FROM_FILE` command to something like `READ_COMMANDS_FROM_FILE` (or something a bit shorter, but still starting with a verb)?
> 
> **EDIT:** Perhaps `IMPORT_COMMAND_FILE` or maybe `IMPORT_COMMANDS`?

I don't think so, `FROM_FILE` sounds pretty fine and intuitive.

Reply:
> > > Others remote commands start with a verb, so I think `CLEAR_LOGS` would be more consistent (and plural because it clears all logs).
> > 
> > 
> > In that case, should we also be renaming the `FROM_FILE` command to something like `READ_COMMANDS_FROM_FILE` (or something a bit shorter, but still starting with a verb)?
> > **EDIT:** Perhaps `IMPORT_COMMAND_FILE` or maybe `IMPORT_COMMANDS`?
> 
> I don't think so, `FROM_FILE` sounds pretty fine and intuitive.

I tend to agree, `FROM_FILE` works for me.